modules/docs/sc_param.md

```diff
@@ -36,76 +36,76 @@ The sc_param module provides methods to help you handle parameters for your stre
+| output_line_break_replacement_character | string | " "                                                                           | replace all replace line break with this parameter value in the output (default value is a blank space)                                                                                                                                                                                                                                       | service_status(neb), host_status(neb)                                                                                                                                                                                                                                                    |                                                                                                                                                                                                                                                                                                  |
+| output_size_limit                       | number | ""                                                                            | put a character number limit for the output (no limit by default)                                                                                                                                                                                                                                                                             | service_status(neb), host_status(neb)                                                                                                                                                                                                                                                    |                                                                                                                                                                                                                                                                                                  |
```

Comment:
if your output is longer than this number, it will be truncated?

Reply:
yes it is!

Reply:
can we add this sentence?

Reply:
done